src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -1738,6 +1741,10 @@ public boolean isUnixSlave() {
         return amiType.isUnix();
     }
 
+    public boolean isMacSlave() {
```

Comment:
We should avoid the use of the world slave in new apis could this be amended to be 
```suggestion
    public boolean isMacAgent() {
```